engines/twp/time.cpp

```diff
@@ -0,0 +1,52 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#define FORBIDDEN_SYMBOL_ALLOW_ALL
```

Comment:
Please, rewrite this and use `g_system->getTimeAndDate()`, instead of direct calls to `localtime()`

Reply:
This is what I wanted to do **but** there is one method missing: how I can get the matching `int64` representing the value returned by `time()`  (value that approximates the number of seconds that have elapsed since the Epoch).
Why do I need this value? Because this is the one saved in the original game and I want to keep the compatibility with the original game.